--- pgaccess-0_99_0_20040219/pgaccess.tcl.orig	2004-04-13 00:52:14.362637336 +0200
+++ pgaccess-0_99_0_20040219/pgaccess.tcl	2004-04-13 00:53:23.092188856 +0200
@@ -740,7 +740,7 @@
         # now that pgtcl is a proper tcl package, try it
         if {[catch {package require Pgtcl} msg]} {
             debug "\tCouldn't use the new Pgtcl package."
-			set shlib [file join $PgAcVar(PGLIB) libpgtcl][info sharedlibextension]
+			set shlib [file join $PgAcVar(PGLIB) libpgtcl].so.2
 			if {![file exists $shlib]} {
 				debug "\nError: Shared library file: '$shlib' does not exist. \n\
 					Check this file, or check PGLIB variable (in pgaccess.cfg)\n"
