--- pgaccess-0_99_0_20040219/pgaccess.tcl.orig	2004-02-20 03:08:02.000000000 +0100
+++ pgaccess-0_99_0_20040219/pgaccess.tcl	2004-04-13 00:51:41.308662304 +0200
@@ -235,22 +235,7 @@
 
 	# the script's home dir
 	if {![info exists PgAcVar(PGACCESS_HOME)]} {
-        set tmphome [info script]
-        if {[file type $tmphome] == "link"} {
-            set home [file dirname [file readlink $tmphome]]
-        } else {
-            set home [file dirname [info script]]
-        }
-		switch [file pathtype $home] {
-			absolute {set PgAcVar(PGACCESS_HOME) $home}
-			relative {set PgAcVar(PGACCESS_HOME) [file join [pwd] $home]}
-			volumerelative {
-				set curdir [pwd]
-				cd $home
-				set PgAcVar(PGACCESS_HOME) [file join [pwd] [file dirname [file join [lrange [file split $home] 1 end]]]]
-				cd $curdir
-			}
-		}
+		set PgAcVar(PGACCESS_HOME) /usr/share/pgaccess
 		debug "\tPGACCESS_HOME: $PgAcVar(PGACCESS_HOME)"
 	}
 	
@@ -695,7 +680,7 @@
 				if {[info exists PgAcVar(PGACCESS_HOME)]} {
 					puts $fid "# PGACCESS_HOME = $PgAcVar(PGACCESS_HOME)"
 				} else {
-					puts $fid "# PGACCESS_HOME = /usr/lib/pgaccess"
+					puts $fid "# PGACCESS_HOME = /usr/share/pgaccess"
 				}
 				puts $fid ""
 				close $fid
